package.json

```diff
@@ -33,8 +33,11 @@
     "docs:gh-pages": "npm run docs:build && npm run ngh -- --dir ./docs/dist",
     "build": "ng build",
     "build:wp": "npm run build -- playground-wp --prod --aot",
-    "build:package": "npm-run-all -s clean:tmp:lib gulp build:ts build:schematics:package build:inline-resources build:bundle clean:tmp",
-    "build:ts": "tsc -p tsconfig.publish.json && ngc -p tsconfig.publish.json",
+    "build:package": "npm-run-all -s clean:tmp:lib gulp build:ts build:schematics:package build:inline-resources build:move-es2015-bundle build:bundle clean:tmp",
+    "build:ts": "npm-run-all -p build:ts:es5 build:ts:es2015",
+    "build:ts:es5": "ngc -p tsconfig.publish.json",
+    "build:ts:es2015": "ngc -p tsconfig.publish.json -t ES2015 -outDir src/.lib/es2015",
```

Comment:
should we also add a test run pointing to es6 bundle version?

Reply:
Added in [558e27f](/akveo/nebular/pull/1249/commits/558e27f02ced6008975f681f0f7eb89d27eb88bf)